index.js

```diff
@@ -6,23 +6,131 @@
 
 var program = require('commander');
 var colors = require('colors');
+var OnboardingCli = require("./onboardingCli");
+var TranslatorCli = require("./translatorCli");
+var fs = require('fs');
+var helpers = require('./helpers');
+var translatorCli = new TranslatorCli();
+
+// set theme 
+colors.setTheme({
+  silly: 'rainbow',
+  header: 'cyan'
+});
 
 program
     .version('1.0.0')
-    .option('-r, --run', 'Run translator')
-    .option('-v, --validate', 'Validate translator')
+    .option('-o --onboarding [Translator Package Name]', 'Do onboarding for specified thing')
+    .option('-h --hub [Hub Package Name]', 'Gets devices for the given hub')
+    
+    .option('-t --translator [Translator Package Name]', 'Do get property for specified thing, requires -p')
+    .option('-i --id [Device id]', 'Device id you want to use')
+    .option('-g --get [RAML property name]', 'Property name to GET for -t')
+    .option('-s --set [RAML property name]', 'Property name to SET for -t')
+    .option('-v --value [value]', 'Stringified JSON value to pass in')
+    .option('-q, --test', 'testing only, do not use')
     .parse(process.argv);
 
-console.log('***************************************************');
 console.log('Open Translators to Things CLI:');
-console.log('See http://www.opentranslatorstothings.org');
-console.log('***************************************************');
-
-if (program.run) {
-    console.log(make_red('Validating lamp schema implementation: NOT IMPLEMENTED YET'));
-} else if (program.validate) {
-    console.log(make_red('Validating lamp schema implementation: NOT IMPLEMENTED YET'));
-} else {
+console.log('');
+
+if (program.test) {
+    // designated to be used for testing out cli scenarios
+}
+
+else if (program.onboarding) {
+    console.log("------ Doing onboarding for %j".header, program.onboarding);
+    
+    var fileName = "./" + program.onboarding + "_onboardingInfo.json";
```

Comment:
This pattern for constructing the fileName is repeated multiple times.  Consider creating a helper function to construct this.